tensorhive/api/api_specification.yml

```diff
@@ -148,3 +170,38 @@ definitions:
         type: string
         format: date-time
       # TODO Add user id, node id
+
+responses:
+  # Example responses
+  # TODO Define more complex behaviour and use these reusable responses where possible (when API begins to stabilize)
+  NotFound:
+    description: Resource was not found
+  Unauthorized:
+    description: Unauthorized
+  InternalError:
+    description: Internal Server Error
+
+parameters:
+  hostnameParam:
+    description: Node's hostname in the network
+    in: path
+    name: hostname
+    required: true
+    type: string
+  gpuMetricTypeQuery:
+    description: Metric type. If not present, queries for all metrics
+    in: query
+    name: metric_type
+    required: false
+    type: string
+    enum:
+      - name
```

Comment:
Maybe these metric names should match the keys in the map returned in the case when no metric is specified, (utilization.gpu [%], power.draw [W]...). Maybe even those keys should be modified, this enum here looks better.

Reply:
Fix in a7839e47b8ede8086dcf107acf328230b98147c4
That's what you had in mind?

Reply:
Exactly